test/skimNtuple_HHbtag.cpp

```diff
@@ -410,6 +410,10 @@ int main (int argc, char** argv)
   int selectedNoWeightsEventsNum = 0 ;
   float totalEvents_PUReweight_up = 0 ;
   float totalEvents_PUReweight_down = 0 ;
+  std::vector<float> totalEvents_QCDscale(7, 0);
+  std::vector<float> totalEvents_pdf(101, 0);
+  std::vector<float> totalEvents_alphaS(2, 0);
```

Comment:
You could set the size the the size of ```theBigTree.MC_QCDscale``` and friends, so in case those change these remain correct.